Classes/Security/DocumentVoter.php

```diff
@@ -491,11 +491,7 @@ protected function canSuggestRestore($document)
             $objectManager =GeneralUtility::makeInstance(ObjectManager::class);
             $documentRepository = $objectManager->get(DocumentRepository::class);
 
-            $linkedDocument = $documentRepository->findOneByLinkedUid($document->getUid());
-
-            if (!$linkedDocument && $document->getObjectIdentifier()) {
-                $linkedDocument = $documentRepository->findOneByLinkedUid($document->getObjectIdentifier());
-            }
+            $linkedDocument = $documentRepository->findOneByLinkedUid($document->getProcessNumber());
```

Comment:
@haogatyp the find method is now using a process number instead of a TYPO3 UID. Would it be safe to rename the method, or is the name coupled with a data base field?